### PR TITLE
Update api documentation 1.5.5-SNAPSHOT

### PR DESCRIPTION
**Pull request by volkflo**
Updated the apidocs with the newest snapshot.

### CODE REVIEW

File: latest/apidoc/manager/font/Droid_Sans/LICENSE.txt [hunk 1]
```diff
@@ -1,202 +1,202 @@
-
```
Comment (msavy) [inline latest/apidoc/manager/font/Droid_Sans/LICENSE.txt]: What's going on with these license files? Are they just indentation changes or something? If so, I think it's better just to leave that out.

File: latest/apidoc/manager/input/input.js [hunk 2]
```diff
@@ -6,8 +6,8 @@ var com = {
 };
 com.qmino.miredot.restApiSource = {
```
Comment (msavy) [inline latest/apidoc/manager/input/input.js]: I see stuff has been reordered here by the generator which is hard to compare (typical issue!), but I presume you've had a good play around with it and everything looks good/same as . before?

File: latest/apidoc/manager/font/Droid_Sans/LICENSE.txt (hunk 1 above)
Comment (msavy) [reply latest/apidoc/manager/font/Droid_Sans/LICENSE.txt]: Or is it autogenerated? In which case it probably doesn't matter too much.

Comment (volkflo) [reply latest/apidoc/manager/font/Droid_Sans/LICENSE.txt]: I am not sure, I didn't actively changed the indentation of this files. I will push a fixup and remove these.

File: latest/apidoc/manager/input/input.js (hunk 2 above)
Comment (volkflo) [reply latest/apidoc/manager/input/input.js]: Yes, it is mostly the same, some new fields are added, but it is is working fine.

**Comment by volkflo** [comment]
I removed the license files. Did not see any difference between the old and new version.

**Pull request merged**